trie/node.go

```diff
@@ -279,3 +298,26 @@ func shouldStopIfContextDone(ctx context.Context, idleProvider IdleNodeProvider)
 		}
 	}
 }
+
+func getNodeBytesForSnapshot(db pruningStorer, childHash []byte) ([]byte, bool, error) {
+	epoch, err := db.GetLatestStorageEpoch()
+	if err != nil {
+		return nil, false, err
+	}
+
+	if epoch == 0 {
+		return nil, false, fmt.Errorf("invalid epoch for getChildForSnapshot")
```

Comment:
getNodeBytesForSnapshot in error message?

Reply:
Done.